fs/testdata/copy_file_bom.txt

```diff
@@ -0,0 +1,2 @@
+0x00, 0x00, 0xFE, 0xFF
```

Comment:
I couldn't see BOM in this file (or am I missing something..?)

<img width="780" alt="Screenshot 2025-02-20 at 15 10 09" src="https://github.com/user-attachments/assets/d0125fd7-1292-4fe7-8c3b-d60b963085a0" />

You can create a text file with BOM like:

```js
Deno.writeTextFileSync("hello.txt", "\ufeffhello");
```

Reply:
Update. I made a mistake about the BOM.